core/templates/dev/head/domain/exploration/ParamChangesObjectFactorySpec.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview unit tests for ParamChanges object factory.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { ParamChangesObjectFactory } from
```

Comment:
Add a newline above to separate Oppia imports from the Angular's imports.

Reply:
I have made the suggested changes in the new commit